activesupport/lib/active_support/logger_thread_safe_level.rb

```diff
@@ -4,10 +4,12 @@
 require "active_support/core_ext/module/attribute_accessors"
 require "concurrent"
 require "fiber"
+require "logger"
 
 module ActiveSupport
   module LoggerThreadSafeLevel # :nodoc:
     extend ActiveSupport::Concern
+    include Logger::Severity
```

Comment:
Is this necessary? The patch to `7-0-stable` did not include it: https://github.com/rails/rails/commit/6a7c72c078715380bf24f029624e6b1fb3035c97